material_3_demo/README.md

```diff
@@ -1,16 +1,34 @@
-# material_3_demo
+# Material You Demo
```

Comment:
I feel like we should rename the directory to `material_you_demo` to match the name